docs/docs/how-tos/nebari-gcp.md

```diff
@@ -174,6 +174,18 @@ With the `qhub-config.yaml` configuration file now created, Nebari can be deploy
 qhub deploy -c qhub-config.yaml
 ```
 
+:::note
+If you have account on any [cloud provider](https://www.nebari.dev/getting-started/cloud-providers) you can create a domain name of your choice. Taking Cloud Flare as a choice you can create a domain name with some restrictions obviously, ***for eg: any-domain-name.qhub.dev***. please keep in mind that after creating your own domain name you need to run  
```

Comment:
As we are already talking about the DNS records in the steps below, I would suggest splitting this note into two things:
1. Sumarize the use of `--dns-provider` and `--dns-autoprovision` flags, suggesting its use for automatic DNS record creation, which will then require the extra environment variable to be set.
2. The second step would be to remove the 
```
Please
follow the instructions in the [Nebari DNS](/how-tos/domain-registry.md) section for more information regarding the domain `A/CNAME` records and how to automatically generate them.
```
from below, and put it in this note.

Reply:
Do I need to still add the use of `--dns-provider` and `--dns-autoprovision` flags? As I have already added the [link](https://www.nebari.dev/how-tos/nebari-gcp) to the note section in the commit. Please let me know @viniciusdc. Thanks.